taxonomies/ingredients.txt

```diff
@@ -39610,14 +39648,17 @@ nl:geconcentreerd aardbeiensap
 # 58 products @2018-10-01
 
 <en:strawberry
-fr:fraises lyophilisées
+en:freeze-dried strawberries
 de:Erdbeeren gefriergetrocknet
+fr:fraises lyophilisées
 fi:pakkaskuivattu mansikka, pakastekuivattu mansikka
 it:fragole liofilizzate
 # openfoodfacts:https://world.openfoodfacts.org/ingredient/fr:fraises-lyophilisées
 # 38 products in 3 languages @2108-10-01
 
-<fr:fraises lyophilisées
+<en:freeze-dried strawberries
+en:freeze-dried strawberry pieces
+de:gefriergetrocknete Erdbeerstücke, gefriergetrocknete Erdbeerscheiben
```

Comment:
These are also handled by the processing taxonomy if you add `Erdbeer` as a synonym for strawberry in the ingredients taxonomy and `scheiben` to the processing taxonomy.

Reply:
to "en:sliced"? "Scheiben" means "slices"

Reply:
en:sliced sounds good